tasks.rb

```diff
@@ -3501,6 +3511,8 @@ def set_hpxml_rim_joists(hpxml_file, hpxml)
                          solar_absorptance: 0.7,
                          emittance: 0.92,
                          insulation_assembly_r_value: 23.0)
+  elsif ['base-bldgtype-single-family-attached.xml'].include? hpxml_file
+    hpxml.rim_joists[-1].area = 67
```

Comment:
If we are changing the sample files, that should be done in a separate PR against master.

Reply:
Yeah, I'll make it there too if you deem these are good changes.